src/protocol/control_layer/ControlMessage.js

```diff
@@ -41,14 +41,47 @@ class ControlMessage {
     }
 
     static registerClass(version, type, clazz) {
+        if (classByType[version] === undefined) {
+            classByType[version] = {}
+        }
         classByType[version][type] = clazz
     }
 
     static getClass(version, type) {
-        if (version !== 0 && version !== 1) {
+        const classesByVersion = classByType[version]
+        if (!classesByVersion) {
             throw new UnsupportedVersionError(version, 'Supported versions: [0, 1]')
```

Comment:
Set of supported versions is available as keys in `classByType`. If we construct the error message dynamically, we don't need to update the hard-coded error message on new version.